composer.json

```diff
@@ -37,10 +37,8 @@
     "require": {
         "php": ">=7.4",
         "ext-curl": "*",
-        "ext-json": "*",
         "ext-zip": "*",
```

Comment:
What if one day they're not provided by core? They are explicit dependencies of this module so, IMHO, they should be declared in the composer file. @greg-1-anderson @kporras07 @namespacebrian how do you guys feel about this?

Reply:
FYI Drupal 10 is going to require JSON storage in the database, so the requirement on JSON is going to get more important, not less.

Reply:
In theory, Drupal core cannot drop a requirement or bump up to a higher major version of guzzle without also bumping its major version (that is, one should expect that the major version of guzzle should not change until D10). So, in theory, it's fine to remove these constraints here.

However, if in practice Drupal did make such a change, the explicit dependency on guzzle here would prevent the use of this module with an incompatible version of guzzle. Since the inclusion of the explicit dependency here is not causing any harm, as far as I can tell, my inclination is to say we're probably better off leaving them in place, as documentation if for no other reason.

Reply:
Ok. I think I'm voting to leave both of these in the composer file, so going to close this without merging.